lib/elixir/lib/process.ex

```diff
@@ -239,10 +239,18 @@ defmodule Process do
   which is not alive or when the given PID exits. Note that timers will not be
   automatically canceled when `dest` is an atom (as the atom resolution is done
   on delivery).
-  """
-  @spec send_after(pid | atom, term, non_neg_integer) :: reference
-  def send_after(dest, msg, time) do
-    :erlang.send_after(time, dest, msg)
+
+  The function accepts one boolean option: `:abs`. When `false` (the default),
+  `time` is treated as relative to the current monotonic time. When `true`
+  `time` is an absolute value of the erlang monotonic time when `msg` should
+  be delivered to `dest`. The concept of the erlang monotonic time (and other time
+  measures kept by the VM) can be explored in the
+  [erlang documentation](http://erlang.org/doc/apps/erts/time_correction.html#Erlang_Monotonic_Time).
```

Comment:
Instead of this section, I would prefer the standard `## Options` section we have when there are options, something like this:

```elixir
@doc """
...

## Options

  * `:abs` - (boolean) when `false`, `time` is treated as relative to the current monotonic time. When `true`, `time` is the absolute value of the Erlang monotonic time at which `msg` should be delivered to `dest`. To read more about Erlang monotonic time and other time-related concepts, look at the documentation for the `System` module.
"""
```

Wdyt?